src/main/java/org/jpeek/Main.java

```diff
@@ -39,15 +39,12 @@
  *
  * <p>There is no thread-safety guarantee.
  *
- * @author Yegor Bugayenko (yegor256@gmail.com)
- * @version $Id$
  * @since 0.1
  * @todo #308:30min Due to introduce new version of qulice 0.18.17 there were
```

Comment:
@dgarus I think you need to remove this todo, before the PR is merged.

Reply:
Ok